src/qml/DigitizingToolbar.qml

```diff
@@ -161,32 +161,26 @@ VisibilityFadingRow {
 
   QfToolButton {
     id: addVertexButton
-    iconSource: Theme.getThemeIcon( "ic_add_vertex_white_24dp" )
     round: true
     enabled: !screenHovering
     bgcolor: {
-        if (!showConfirmButton)
+        if (!enabled)
+          Theme.darkGraySemiOpaque
+        else if (!showConfirmButton)
           Theme.darkGray
         else if (Number( rubberbandModel ? rubberbandModel.geometryType : 0 ) === Qgis.GeometryType.Point ||
                  Number( rubberbandModel.geometryType ) === Qgis.GeometryType.Null)
           Theme.mainColor
         else
           Theme.darkGray
```

Comment:
```suggestion
        if (!enabled) {
          Theme.darkGraySemiOpaque
        } else if (!showConfirmButton) {
          Theme.darkGray
        } else if (Number( rubberbandModel ? rubberbandModel.geometryType : 0 ) === Qgis.GeometryType.Point ||
                 Number( rubberbandModel.geometryType ) === Qgis.GeometryType.Null) {
          Theme.mainColor
        } else {
          Theme.darkGray
        }
```
For complex if/else conditions lack of {} makes it hard to read.

Reply:
@suricactus , done.